themes/he/layouts/partials/header.html

```diff
@@ -80,7 +82,8 @@
           />
         </svg>
       </button>
-
+      {{end}}{{end}}

```

Comment:
@arunkumar295 Check why 2 ends are added instead od one.

Reply:
one end is for if condition, I have given  and another one is header starting condition .I replaced it as it is before .